test/mini_sql/mysql/builder_test.rb

```diff
@@ -57,45 +45,6 @@ def test_append_params
     assert_equal(0, builder.exec)
   end
 
-  def test_offset_limit
```

Comment:
why are all these tests being removed?

Reply:
the number of tests has not changed, these tests are full duplicates in `test/mini_sql/builder_tests.rb`
they have been overridden since mysql does not support `create temp table`, now I have replaced it with `create temporary table` and it works in all databases